src/pages/workspace/members/ImportMembersPage.tsx

```diff
@@ -0,0 +1,21 @@
+import type {StackScreenProps} from '@react-navigation/stack';
+import React from 'react';
+import ImportSpreedsheet from '@components/ImportSpreadsheet';
+import type {SettingsNavigatorParamList} from '@libs/Navigation/types';
+import ROUTES from '@src/ROUTES';
+import type SCREENS from '@src/SCREENS';
+
+type ImportMembersPageProps = StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.MEMBERS_IMPORT>;
+
+function ImportMembersPage({route}: ImportMembersPageProps) {
+    const policyID = route.params.policyID;
+
+    return (
+        <ImportSpreedsheet
```

Comment:
We need to wrap it with `AccessOrNotFoundWrapper` to avoid issue https://github.com/Expensify/App/issues/51129